src/components/Starfield.astro

```diff
@@ -9,7 +9,7 @@ const { subtle, noGradient } = Astro.props as Props;
 
 <div class:list={[
     "pointer-events-none",
-    "absolute inset-0 -z-50 border-t-[3.5rem] border-slate-900",
```

Comment:
Looks like removing the negative `z-index` here means the starfield covers text in the page header on other pages, e.g. https://deploy-preview-497--astro-www-2.netlify.app/blog/

Maybe instead of removing here you could set a lower `z-index` in the homepage Hero component?

Reply:
Ah yes, thanks for catching this xD
I tried setting the lower `z-index` for the Hero component, but it breaks the HeroCard interactions on the homepage.

I'll see what else I can try to get this working.

Reply:
As I published that update, I found a fix by instead increasing the `z-index` on that same Hero component. As far as I could see, everything now plays nicely and stacks in the correct order

<details>
<summary>Screenshots</summary>
<br/>
Homepage
<br/>
<img src="https://user-images.githubusercontent.com/64310361/211168586-1987fcfc-5a99-4361-9116-6275f1df8d88.png" alt="" />
<br/>
Other page headers (e.g. blog)
<br/>
<img src="https://user-images.githubusercontent.com/64310361/211168591-b5bf3472-1f83-433b-b3e8-29a2950f9603.png" alt="" />
<br/>
Lighthouse score
<br/>
<img src="https://user-images.githubusercontent.com/64310361/211168629-f6a4b898-ed12-47fe-b6a6-9cdff02b2b29.png" alt="" />
</details>